appf.adoc

```diff
@@ -433,7 +433,9 @@ to a reference direction.
 |  Applied to all abscissa values in the rectangular
     coordinates for a map projection in order to eliminate negative numbers. Expressed in
     the unit of the coordinate variable identified by the
-    standard name **`projection_x_coordinate`**.
+    standard name **`projection_x_coordinate`**. If 
```

Comment:
@neumannd could you move this sentence onto its own line (and on line 449) so it's one sentence, one line?

In source code it makes sense to restrict line length, but for a document like the Conventions it makes it easier to compare changes if each sentence stands on its own line. Then a blame doesn't produce false positives. We don't need to reflow all the text in the documents but when we touch or add a sentence I think it's best to put them on their own line.